src/python/foglamp/core/api/scheduler.py

```diff
@@ -336,9 +336,9 @@ def _extract_args(data, curr_value):
             'name': sch.name,
             'process_name': sch.process_name,
             'type': Schedule.Type(int(sch.schedule_type)).name,
-            'repeat': str(sch.repeat),
+            'repeat': sch.repeat.total_seconds() if sch.repeat else 0,
+            'time': (sch.time.hour * 60 * 60 + sch.time.minute * 60 + sch.time.second) if sch.time else 0,
             'day': sch.day,
-            'time': str(sch.time),
```

Comment:
is `server.Server.scheduler.` okay to receive time and repeat? don;t we need to convert to python timedelta struct?